File: 2025_03/프로그래머스 - Lv.1/과일 장수/과일 장수.js
// 내림차순 정렬 -> 큰거부터 묶기
function solution(k, m, score) {
  score.sort((a, b) => b - a);
  let answer = 0;
  for(let i = 0 ; i < score.length ; i += m){
    const box = score.slice(i, i + m);
    const min = box[m-1];
    if(min) answer += m * min;
  }
  return answer;
}

File: 2025_03/프로그래머스 - Lv.1/과일 장수/과일 장수.py
# 내림차순 정렬 -> 큰거부터 묶기
def solution(k, m, score):
    answer = 0
    score.sort(reverse=True)
    for i in range(0, len(score), m):
        box = score[i:i+m]
        if len(box) == m:
            answer += m * min(box)
    return answer